textdb/textdb-web/src/main/java/edu/uci/ics/textdb/web/request/QueryPlanRequest.java

```diff
@@ -22,7 +23,9 @@
     private ArrayList<OperatorBean> operatorBeans;
     @JsonProperty("links")
     private ArrayList<OperatorLinkBean> operatorLinkBeans;
+    @JsonIgnore
     private HashMap<String, HashMap<String, String>> operatorProperties;
+    @JsonIgnore
```

Comment:
It ignores the field when serializing to JSON. We don't want the `LogicalPlan` object and the `operatorProperties` to be serialized to JSON because they are used internally and don't need to be exposed when returning a JSON from the `PlanStore`